src/view/utils/point.js

```diff
@@ -19,4 +19,78 @@ const formatDateForEditPoint = (date) => (date !== null ? dayjs(date).format('D/
 
 const getTotalDate = (dateFrom, dateUntil) => `${dayjs(dateFrom).format('MMM D')} - ${dayjs(dateUntil).format('D')}`;
 
-export { getTimeDifference, formatDateForEditPoint, getTotalDate };
+const getWeightNullDate = (firstDate, secondDate) => {
+  if (firstDate === null && secondDate === null) {
+    return 0;
+  }
+
+  if (firstDate === null) {
+    return 1;
+  }
+
+  if (secondDate === null) {
+    return -1;
+  }
+
+  return null;
+};
+
+const sortDay = (firstPoint, secondPoint) => {
+  const weight = getWeightNullDate(firstPoint.dateFrom, secondPoint.dateFrom);
+
+  if (weight !== null) {
+    return weight;
+  }
+
+  return dayjs(firstPoint.dateFrom).diff(dayjs(secondPoint.dateFrom));
+};
+
+const sortEvent = (firstPoint, secondPoint) => {
+  if (firstPoint.type > secondPoint.type) {
+    return 1;
+  }
+
+  if (firstPoint.type < secondPoint.type) {
+    return -1;
+  }
+
+  return 0;
+};
+
+const sortTime = (firstPoint, secondPoint) => {
+  if (dayjs(firstPoint.dateUntil).diff(dayjs(firstPoint.dateFrom)) < dayjs(secondPoint.dateUntil).diff(dayjs(secondPoint.dateFrom))) {
+    return 1;
+  }
+
+  if (dayjs(firstPoint.dateUntil).diff(dayjs(firstPoint.dateFrom)) > dayjs(secondPoint.dateUntil).diff(dayjs(secondPoint.dateFrom))) {
+    return -1;
+  }
+
+  return 0;
+};
+
+const sortPrice = (firstPoint, secondPoint) => {
+  if (firstPoint.price < secondPoint.price) {
+    return 1;
+  }
+
+  if (firstPoint.price > secondPoint.price) {
+    return -1;
+  }
+
+  return 0;
+};
```

Comment:
```suggestion
const sortPrice = (firstPoint, secondPoint) => {
  return firstPoint.price > secondPoint.price
};
```
Почти все можно переделать так 😁